net/tcp/impl/posix.h

```diff
@@ -122,9 +153,15 @@ class SocketHandle {
 
  private:
   SocketHandle() = delete;
-  SocketHandle(const SocketHandle&) = delete;
   void operator=(const SocketHandle&) = delete;
   void operator=(SocketHandle&&) = delete;
+
+#ifndef BRICKS_WINDOWS
+  SocketHandle(const SocketHandle&) = delete;
+#else
+ public:
```

Comment:
Visibility of any member following this `#define` will depend on `BRICKS_WINDOWS`.